target/linux/bcm63xx/dts/bcm6328-technicolor-tg582n.dts

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
```

Comment:
dts-v1 has been moved to dtsi and can be dropped here.

Reply:
Removed. I'll remove it to the other PRs too